tavla/app/components/ContactForm.tsx

```diff
@@ -72,6 +73,14 @@ function ContactForm() {
                         med Tavla.
                     </Paragraph>
 
+                    <ClientOnlyTextField
+                        label="E-postadresse"
+                        name="email"
+                        id="email"
+                        aria-label="E-postadresse"
+                        {...getFormFeedbackForField('email', formState)}
+                    />
+
                     <div>
```

Comment:
to textfields 👀 
<img width="568" alt="image" src="https://github.com/user-attachments/assets/be4d5359-00fc-410c-a4db-73cdb929d2c6">

Reply:
Den øverste skal fjernes